distribution/docs/src/main/jdocs/content/_tables/org.codice.ddf.ui.search.standard.endpoint-table-contents.adoc

```diff
@@ -26,5 +26,12 @@
 |false
 |false
 
+|Set Max Query Results
+|setMaxQueryResults
```

Comment:
✏️  Looks like the name of this is actually `Maximum Query Results`/`maxQueryResults` (without the set part). Change this to reflect what is in the blueprint for consistency.